packaging/standalone/permalinks.properties

```diff
@@ -1,20 +1,14 @@
 neo4j-home.url=http://neo4j.com/
 neo4j-home.url.title=Neo4j Home
-neo4j-manual.url=http://neo4j.com/docs/#{project.version}/
+neo4j-manual.url=http://neo4j.com/docs/
 neo4j-manual.title=The Neo4j Manual
```

Comment:
The link no longer leads to The Manual, maybe change the title to "Neo4j Documentation" or similar.